### DISCUSSION TITLE
`app.organicmaps` vs `app.organicmaps.web` ?

### DISCUSSION THREAD

**Discussion by KaKi87** [organicmaps/organicmaps#9261] [Q&A]
Hi,

What's the difference between those two ?

![](https://github.com/user-attachments/assets/8a5d772d-8712-4c56-8496-7abad2b6adee)
![](https://github.com/user-attachments/assets/1b71d37d-ee3c-4166-868a-083aeb937b55)

Thanks

**Comment by rusty-snake** [comment]
See https://github.com/orgs/organicmaps/discussions/7719

One is coming from your F-Droid repo and one from your IzzyOnDroid repo.

**Reply by KaKi87** [reply]
> Available instantly.

Yet late on IzzyOnDroid compared to 
F-Droid ?
